conda/environments/cuspatial_dev_cuda11.0.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - cudf=0.20.*
   - cudatoolkit=11.0
   - gdal>=3.0.2
-  - geopandas=0.7.0
+  - geopandas>=0.9.0
```

Comment:
looks like this change will also require the version spec in the `integration` repo to be increased.
@thomcom, can you open that PR?

https://github.com/rapidsai/integration/blob/branch-0.20/conda/recipes/versions.yaml#L73-L74

Reply:
Thanks @ajschmidt8, here it is: https://github.com/rapidsai/integration/pull/271